quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/Classes.java

```diff
@@ -14,10 +14,15 @@
 package co.paralleluniverse.fibers.instrument;
 
 import java.lang.reflect.UndeclaredThrowableException;
-import java.util.ArrayList;
 import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Set;
+import java.util.HashSet;
+import java.util.EnumMap;
 
+import co.paralleluniverse.fibers.Instrumented;
+import co.paralleluniverse.fibers.Suspendable;
```

Comment:
The `instrument` package _cannot_ reference the `fibers` package. This should have generated some OSGi warnings / errors!

Reply:
Yes it did, see above. This was why I probably incorrectly excluded classes from the build.gradle file. I get this error with the exclude line taken out.
`
* What went wrong:
Execution failed for task ':quasar-core:validateAgentBundle'.
> OSGi bundle for Java agent should only import from itself: co.paralleluniverse.common.resource,co.paralleluniverse.fibers
`